project/Dependencies.scala

```diff
@@ -111,6 +111,8 @@ object Dependencies {
   val databaseSqlDependencies = baseDependencies ++ slickDependencies ++ dbmsDependencies
 
   val coreDependencies = List(
+    "org.scala-lang" % "scala-compiler" % Settings.ScalaVersion % Provided,
+    "eu.timepit" %% "refined"            % "0.7.0",
```

Comment:
what's up w/ the spacing here. if you're trying to line stuff up (and nothing else is), it's not lined up right.